frontend/src/index.css

```
@@ -1789,3 +1789,154 @@ tbody tr:hover td {
+  background: linear-gradient(90deg, #b45309, #92400e);
+  color: #fef3c7;
+  font-size: 0.875rem;
+  font-weight: 500;
+  text-align: center;
+  justify-content: center;
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.4);
+  /* Animation */
+  animation: slideDown 0.25s ease-out;
+}
+
+@keyframes slideDown {
```

Stylelint (17.14.1)

[error] Expected keyframe name "slideDown" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)